### DISCUSSION TITLE
what is the meaning of red boxplot in  plot.subsample？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#143] [Q&A]
what is the meaning of red boxplot in  plot.subsample？

![image](https://user-images.githubusercontent.com/5112397/147751094-0a723f19-e036-41b5-88ab-d088a362b724.png)

library(randomForestSRC)
reg.o <- rfsrc(mpg ~ ., mtcars)
reg.smp.o <- subsample(reg.o, B = 25, subratio = .5)
plot.subsample(reg.smp.o)

**Comment by ishwaran** [comment]
The red box plots are the variables with VIMP significantly different from zero at the specified `alpha` value.  The blue are the non-significant variables.
